.mci.yml

```diff
@@ -151,6 +153,32 @@ buildvariants:
       tasks:
           - name: compile_and_test
 
+    - name: ubuntu1404-debug-boost
+      display_name: "Ubuntu 14.04 Debug with boost"
```

Comment:
for brevity, maybe just "Ubuntu 14.04 Debug (boost)" and "Ubuntu 14.04 Debug (experimental)".  In some views mci doesn't show long names well.

I'd also be ok with an even shorter version, like "Ubuntu (boost)" and "Ubuntu (experimental)."  Of course with this you'd have to go look in .mci.yml to see the ubuntu version, but meh.  I leave it to your discretion.